pyproject.toml

```
@@ -1,5 +1,66 @@
 [build-system]
-requires = [ "setuptools>=61", "setuptools-scm>=8" ]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "cfgrib"
+dynamic = ["version"]
+description = "Python interface to map GRIB files to the NetCDF Common Data Model following the CF Convention using ecCodes."
+readme = "README.rst"
+license = "Apache-2.0"
+requires-python = ">=3.7"
+keywords = ["eccodes", "grib", "xarray"]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: Apache Software License",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+dependencies = ["attrs>=19.2", "click", "eccodes>=0.9.8", "numpy"]
+
+[[project.authors]]
+name = "European Centre for Medium-Range Weather Forecasts (ECMWF)"
+email = "software.support@ecmwf.int"
+
+[project.optional-dependencies]
+tests = [
+  "dask[array]",
+  "flake8",
+  "pytest",
+  "pytest-cov",
+  "scipy",
+  "xarray>=0.15",
+]
+xarray = ["xarray>=0.15"]
+
+[project.scripts]
+cfgrib = "cfgrib.__main__:cfgrib_cli"
+
+[project.entry-points."xarray.backends"]
+cfgrib = "cfgrib.xarray_plugin:CfGribBackend"
+
+[project.urls]
+Homepage = "https://github.com/ecmwf/cfgrib"
+
+[tool.hatch.version]
+path = "cfgrib/__init__.py"
+
+[tool.hatch.build.targets.sdist]
+include = ["/cf2cdm"]
```

```suggestion
[tool.hatch.build.targets.wheel]
include = ["cf2cdm", "cfgrib"]
```

I ran into problems attempting to build this branch, and the suggestion hatch printed looked a lot like the change above

---

I have added both `cf2cdm` and `cfgrib` to `sdist` and `wheel` targets now.